### PR TITLE
update ZScale ZIA readme

### PR DESCRIPTION
**Pull request by ellis-elastic**
added link to Zscaler documentation and improved the wording of the other links

## What does this PR do?

added link to Zscaler documentation and improved the wording of the other links

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Review by bmorelli25 — APPROVED** [review]

**Comment by bmorelli25** [comment]
It looks like something might've gone wrong with your merge.

```
[2022-05-04T10:52:33.651Z] README.md is outdated. Rebuild the package with 'elastic-package build'
```

File: packages/zscaler_zia/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 
 ## Steps for setting up NSS Feeds
 
 1. Enable the integration with the TCP input.
-2. Configure the Zscaler NSS Server and NSS Feeds to send logs to the Elastic Agent that is running this integration. See [_Add NSS Server_](https://help.zscaler.com/zia/adding-nss-servers) and [_Add NSS Feeds_](https://help.zscaler.com/zia/adding-nss-feeds). Use the IP address hostname of the Elastic Agent as the 'NSS Feed SIEM IP Address/FQDN', and use the listening port of the Elastic Agent as the 'SIEM TCP Port' on the _Add NSS Feed_ configuration screen. To configure Zscaler NSS Server and NSS Feeds follow the following steps.
+2. Configure the Zscaler NSS Server and NSS Feeds to send logs to the Elastic Agent that is running this integration. See [Add NSS Server](https://help.zscaler.com/zia/adding-nss-servers) and [Add NSS Feeds](https://help.zscaler.com/zia/adding-nss-feeds). Use the IP address hostname of the Elastic Agent as the 'NSS Feed SIEM IP Address/FQDN', and use the listening port of the Elastic Agent as the 'SIEM TCP Port' on the _Add NSS Feed_ configuration screen. To configure Zscalar NSS Server and NSS Feeds follow the following steps.
```
Comment (taylor-swanson) [inline packages/zscaler_zia/_dev/build/docs/README.md:11]:
```suggestion
2. Configure the Zscaler NSS Server and NSS Feeds to send logs to the Elastic Agent that is running this integration. See [Add NSS Server](https://help.zscaler.com/zia/adding-nss-servers) and [Add NSS Feeds](https://help.zscaler.com/zia/adding-nss-feeds). Use the IP address hostname of the Elastic Agent as the 'NSS Feed SIEM IP Address/FQDN', and use the listening port of the Elastic Agent as the 'SIEM TCP Port' on the _Add NSS Feed_ configuration screen. To configure Zscaler NSS Server and NSS Feeds follow the following steps.
```

File: packages/zscaler_zia/_dev/build/docs/README.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
     - In the ZIA Admin Portal, add an NSS Server.
-        - Log in to the ZIA Admin Portal using your admin account.
-        - Add an NSS server. Refer to Adding NSS Servers to set up an [_Add NSS Server_](https://help.zscaler.com/zia/adding-nss-servers) for Web and/or Firewall.
+        - Log in to the ZIA Admin Portal using your admin account. If you're unable to log in, contact Support.
```
Comment (taylor-swanson) [inline packages/zscaler_zia/_dev/build/docs/README.md:13]: Perhaps add a link to the Zscaler support page?

**Comment by bmorelli25** [comment]
```
[2022-06-29T08:32:41.575Z] README.md is outdated. Rebuild the package with 'elastic-package build'
```

**Review by andrewkroh — APPROVED** [review]

**Pull request merged**